### ISSUE TITLE
Getting error like `React is not defined` while using this Input

### ISSUE THREAD

**Issue by Krishna-Vyas** [anday013/react-native-otp-entry#77]
**Describe the bug**
Facing 'React is not defined'  while using this library

**To Reproduce**
`import {OtpInput} from 'react-native-otp-entry';`
`<OtpInput
            numberOfDigits={6}
            focusColor={Const.Colors.theme_yellow}
            focusStickBlinkingDuration={500}
            onTextChange={text => console.log(text)}
            onFilled={text => onKeyBoardPress(text)}
            textInputProps={{
              accessibilityLabel: 'One-Time Password',
            }}
            type="numeric"
            theme={{
              containerStyle: {},
              pinCodeContainerStyle: styles.txtContainer,
              pinCodeTextStyle: styles.txt,
              focusStickStyle: {
                ...styles.txt,
                color: Const.Colors.theme_yellow,
              },
              focusedPinCodeContainerStyle: {
                ...styles.txtContainer,
                borderWidth: 2,
                borderColor: Const.Colors.theme_yellow,
              },
            }}
          />`

**Screenshots**
<img width="376" alt="Screenshot 2024-10-15 at 9 08 37 PM" src="https://github.com/user-attachments/assets/b4a25216-0706-4ed6-88d6-dc0073f2ec4e">

**Environment (please complete the following information):**
 - OS: ios (React Native)
 - Device OS: ios
 - Device version: iPhone 15(simulator)
 - Library version: 1.7.2

**Comment by talhaakbar22** [comment]
Got same error.

**Comment by anday013** [comment]
I'll take a look. In the meantime, can u please provide your env details? Which version of RN or Expo and React are you using?

**Comment by talhaakbar22** [comment]
> I'll take a look. In the meantime, can u please provide your env details? Which version of RN or Expo and React are you using?

**In my package.json**

```
 "react": "^18.0.0-alpha-fd5e01c2e-20210913",
 "react-native": "^0.65.1",
 "react-native-otp-entry": "^1.7.2",
```

also I'm using the node version `v14.21.3`.

**Comment by Krishna-Vyas** [comment]
@anday013  Thanks for response, here is the versions
"react": "17.0.2",
"react-native": "0.68.6",
 "react-native-otp-entry": "^1.7.2",

**Comment by anday013** [comment]
@Krishna-Vyas @talhaakbar22  issue should be fixed in 1.7.3 version :) Please go ahead and try
